app/models/physical_server.rb

```diff
@@ -110,4 +110,8 @@ def v_availability
   def v_host_os
     host.try(:vmm_product).nil? ? N_("") : host.vmm_product
   end
+
+  def kind
```

Comment:
The variable `kind` seems to generic of a name.  Based on usage here (https://github.com/ManageIQ/manageiq-providers-redfish/pull/51/files#diff-dfdfa668b95bc3eeb77a0e321896dfa6R10) would `vendor` or `product` be better.  Open to other suggestions.

cc @agrare @gtanzillo